adam/runtime/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+# install jdk-7
+RUN apt-get update && apt-get install -y openjdk-7-jdk
+
+# copy jar
+RUN mkdir /opt/cgl-docker-lib
```

Comment:
Looks like the most recent convention is to lose the cgl-docker-lib from /opt.  See e.g. https://github.com/BD2KGenomics/cgl-docker-lib/commit/c59e2e02c59b6648a9015be480c06a89a8dbec04